src/enclave/Enclave/Filter.cpp

```diff
@@ -24,11 +24,9 @@ void filter(uint8_t *condition, size_t condition_length,
         std::string("Filter expression expected to return BooleanField, instead returned ")
         + std::string(tuix::EnumNameFieldUnion(condition_result->value_type())));
     }
-    if (condition_result->is_null()) {
-      throw std::runtime_error("Filter expression returned null");
-    }
 
-    bool keep_row = static_cast<const tuix::BooleanField *>(condition_result->value())->value();
+    // If condition_result is NULL, then always return false
+    bool keep_row = static_cast<const tuix::BooleanField *>(condition_result->value())->value() & !condition_result->is_null();
```

Comment:
1. Best to keep a maximum line length of 100 characters to match Spark and the rest of the C++ codebase.
2. Might as well use logical AND here, unless there's a specific reason to use bitwise AND?